OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatisticServiceTest.cs

```diff
@@ -22,24 +23,30 @@ public class StatisticServiceTest
 
         private Mock<IApplicationRepository> applicationRepository;
         private Mock<IWorkshopRepository> workshopRepository;
+        private Mock<IRatingService> ratingService;
```

Comment:
`ratingService` seems to be not ever used outside `SetUp` method. Thus might be moved inside it as local var

Reply:
Same for `logger`

Reply:
Same for `mapper`